docs/docs-ref-autogen/office/office/office.addin.yml

```diff
@@ -55,7 +55,9 @@ items:
       return:
         type:
           - 'office!Office.Addin#onVisibilityModeChanged~0:complex'
-        description: A promise that resolves when the listener is added.
+        description: >-
+          A promise that resolves to a function when the listener is added. The `RemoveEventListener` type is defined
+          with `type RemoveEventListener = () => Promise<void>`<!-- -->. Calling it removes the listener.
```

Comment:
Is it normal for an empty HTML comment like this `<!-- -->` to be inserted before the period?

Reply:
@Rick-Kirkham That seems normal to me but please see the [preview site](https://review.docs.microsoft.com/en-us/javascript/api/office/office.addin?view=common-js&branch=fix-809-get-latest) to verify if it looks like you expect.